docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -3,7 +3,7 @@
 
 box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
-participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":GuestBookParser" as AddressBookParser LOGIC_COLOR
 participant ":DeleteCommandParser" as DeleteCommandParser LOGIC_COLOR
 participant "d:DeleteCommand" as DeleteCommand LOGIC_COLOR
 participant ":CommandResult" as CommandResult LOGIC_COLOR
```

Comment:
Should the below be changed to LogicManager -> GuestBookParser instead of LogicManager -> AddressBookParser?

Reply:
Likewise in the DeleteCommandParser

Reply:
should be